.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: elixir
+elixir:
+  - 1.1.0
+otp_release:
+  - 17.4
+addons:
+  postgresql: "9.4"
+sudo: false # to use faster container based build environment
+env:
+  - ELIXIR_ERL_OPTIONS=""
+  - ELIXIR_ERL_OPTIONS="+T 9"
```

Comment:
it's a copy from https://github.com/elixir-lang/ecto/blob/master/.travis.yml ;) 

Actually, I did not know that this is for scheduler randomization... good to know!